iac-common/src/main/java/org/sonar/iac/common/api/tree/impl/TextRange.java

```diff
@@ -60,4 +64,33 @@ public int hashCode() {
   public String toString() {
     return "[" + start.line() + ":" + start.lineOffset() + "/" + end.line() + ":" + end.lineOffset() + "]";
   }
+
+  public TextRange trimToText(String content) {
```

Comment:
Maybe `trimEndToText`? It was not obvious to me, what this method does, by its name